I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 283, in script
    $ponstosCringe += 1
  File "game/script.rpy", line 283, in <module>
    $ponstosCringe += 1
NameError: name 'ponstosCringe' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 283, in script
    $ponstosCringe += 1
  File "renpy/ast.py", line 922, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "renpy/python.py", line 2218, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 283, in <module>
    $ponstosCringe += 1
NameError: name 'ponstosCringe' is not defined

Windows-10-10.0.19041
Ren'Py 7.4.6.1693
Cringe Compilation 1.0
Sun Jul 25 20:42:16 2021
